tests/server/providers/jellyfin/test_parsers.py

```diff
@@ -0,0 +1,71 @@
+"""Test we can parse Jellyfin models into Music Assistant models."""
+
+import logging
+import pathlib
+from collections.abc import AsyncGenerator
+
+import aiofiles
+import aiohttp
+import pytest
+from aiojellyfin import Artist, Connection, SessionConfiguration
+from mashumaro.codecs.json import JSONDecoder
+from syrupy.assertion import SnapshotAssertion
+
+from music_assistant.server.providers.jellyfin.parsers import parse_album, parse_artist, parse_track
```

Comment:
this line looks too long to me, is the linter not working on the files in tests ?

Reply:
```
ruff format tests/server/providers/jellyfin/test_parsers.py 
1 file left unchanged

ruff check tests/server/providers/jellyfin/test_parsers.py 
All checks passed!
```

Reply:
And come to think of it, [af7b711](https://github.com/music-assistant/server/pull/1397/commits/af7b711fdf75ab5a91b2a3369f5b84f361756203) was because of the lints running on the tests.